### PR TITLE
Enable WebP

### PR DESCRIPTION
**Pull request by amirh**
fixes https://github.com/flutter/flutter/issues/9857

### CODE REVIEW

File: DEPS [hunk 1]
```diff
@@ -21,8 +21,9 @@ vars = {
   'chromium_git': 'https://chromium.googlesource.com',
   'dart_git': 'https://dart.googlesource.com',
   'fuchsia_git': 'https://fuchsia.googlesource.com',
-  'skia_git': 'https://skia.googlesource.com',
   'github_git': 'https://github.com',
+  'libwebp_git': 'https://chromium.googlesource.com/webm',
```
Comment (chinmaygarde) [inline DEPS]: `https://chromium.googlesource.com` is already present in the `var` above. Please just use that var directly below.

File: DEPS [hunk 2]
```diff
@@ -355,6 +356,9 @@ deps = {
   'src/third_party/libjpeg-turbo':
    Var('skia_git') + '/third_party/libjpeg-turbo.git' + '@' + 'debddedc75850bcdeb8a57258572f48b802a4bb3',
 
+  'src/third_party/libwebp':
+   Var('libwebp_git') + '/libwebp.git' + '@' + '0.6.0',
```
Comment (chinmaygarde) [inline DEPS]: `Var('chromium_git') + '/webm/libwebp.git' + '@' + '0.6.0',`

File: lib/ui/painting/codec.cc [hunk 3]
```diff
@@ -69,18 +70,24 @@ fxl::RefPtr<Codec> InitCodec(sk_sp<SkData> buffer, size_t trace_id) {
...
 
   std::unique_ptr<SkCodec> skCodec = SkCodec::MakeFromData(buffer);
   if (!skCodec) {
+    FXL_LOG(ERROR) << "SkCodec::MakeFromData failed";
     return nullptr;
   }
-  if (skCodec->getFrameCount() > 1) {
+  if (skCodec->getFrameCount() > 1
+      // Temporarily disable WebP animations due to:
+      // https://github.com/flutter/flutter/issues/13017
```
Comment (chinmaygarde) [inline lib/ui/painting/codec.cc]: Please add a `// TODO(amirh)` here.

File: tools/gn [hunk 4]
```diff
@@ -70,7 +70,7 @@ def to_gn_args(args):
     gn_args['skia_enable_flutter_defines'] = True # Enable Flutter API guards in Skia.
     gn_args['skia_use_dng_sdk'] = False    # RAW image handling.
     gn_args['skia_use_sfntly'] = False     # PDF handling.
-    gn_args['skia_use_libwebp'] = False    # Needs third_party/libwebp.
+    gn_args['skia_use_libwebp'] = True     # WebP codec.
```
Comment (chinmaygarde) [inline tools/gn]: I dont think you need this anymore.

File: lib/ui/painting/codec.cc (hunk 3 above)
Comment (amirh) [reply lib/ui/painting/codec.cc]: I completely removed this as the Skia fix landed

File: tools/gn (hunk 4 above)
Comment (amirh) [reply tools/gn]: done

**Pull request merged**